app/services/search_field_service.rb

```diff
@@ -50,7 +50,7 @@ class SearchFieldService
     'title_tesim',
     'toc_tesim',
     'uniform_title_tesim',
-    'writing_and_hands_tesim',
+    'hand_note_sim',
```

Comment:
Also add shelfmark_ssi, descriptive_title_tesim, and delivery_ssi, please verify the field names in feed_ursus